### ISSUE TITLE
quantile aggregation operator does not seem to be working correctly

### ISSUE THREAD

**Issue by alexclear** [VictoriaMetrics/VictoriaMetrics#55]
<img width="1393" alt="Screenshot 2019-06-06 at 11 16 41" src="https://user-images.githubusercontent.com/857605/59017850-3edc6000-884d-11e9-86ad-847cec27582d.png">
<img width="1393" alt="Screenshot 2019-06-06 at 11 19 40" src="https://user-images.githubusercontent.com/857605/59017881-4dc31280-884d-11e9-8a02-08819b15231b.png">
Both graphs plot the same time series data for the same host, the former uses a vanilla Prometheus and the latter uses VictoriaMetrics.             
Targets are:                 

```
          - expr: 'quantile(0.95, rate(node_disk_io_time_seconds_total[5m])*100)'
            legend_format: !unsafe "95th percentile"
          - expr: 'quantile(0.50, rate(node_disk_io_time_seconds_total[5m])*100)'
            legend_format: !unsafe "50th percentile"
```

Quantile values differ significantly on the graphs.

**Comment by valyala** [comment]
Confirmed the bug. It will be fixed in the next release

**Comment by valyala** [comment]
@alexclear , could you confirm that the bug is fixed in [v1.18.10](https://github.com/VictoriaMetrics/VictoriaMetrics/releases)?

**Comment by alexclear** [comment]
Works on my side, thank you!
